File: Todos/C/BEE 3170.c
/*
  autor: Vinist021;
  data: 16/02/2025;
  nome: Bolinhas de Natal
*/

#include<stdio.h>

int main()
{
    int B, G;
    int necessario, qtFaltam;

    scanf("%d %d", &B, &G);

    necessario = G / 2;
    qtFaltam = necessario - B;

    if(qtFaltam > 0)
        printf("Faltam %d bolinha(s)\n", qtFaltam);
    else
        printf("Amelia tem todas bolinhas!\n");

    return 0;
}

File: Todos/Python/BEE 3170.py
#   autor: Vinist021;
#   data: 16/02/2025;
#   nome: Bolinhas de Natal;

B = int(input())
G = int(input())

nec = G // 2
qtFaltam = nec - B

if qtFaltam > 0:
    print(f'Faltam {qtFaltam} bolinha(s)')
else:
    print('Amelia tem todas bolinhas!')